en/docs/learn/api-security/oauth2/grant-types/password-grant.md

```diff
@@ -72,7 +82,7 @@ You can obtain an access token by providing the resource owner's username and pa
 
     Replace the `<base64encode(clientId:clientSecret)>` as appropriate.
 
-    For example, use the following cURL command to access the Token API. It generates two tokens as an access token and a refresh token. You can use the refresh token at the time a token is renewed .
+    For example, use the following cURL command to access the Token API. It generates two tokens as an access token and a refresh token. You can use the refresh token at the time a token is renewed. Replace the `<username>` and `<password>` as appropriate.
```

Comment:
Use 'Format' 'Example' tab for curl commands

Reply:
fixed with b151dd4